azext_iot/tests/iothub/configurations/test_iot_config_unit.py

```diff
@@ -75,6 +76,8 @@ def sample_config_edge(set_cwd, request):
         payload = json.dumps(json.loads(read_file_content(ea_v11_eh_v12_path)))
     elif request.param == "ea_v90_eh_v91":
         payload = json.dumps(json.loads(read_file_content(ea_v90_eh_v91_path)))
+    elif request.param == "billable_module":
```

Comment:
This request.param value is not added to the fixture. This condition is not going to be met. Also what is being tested? We need to _at least_ assert the new modulesPurchase payload is being sent in the request when creating an edge deployment with billable modules content.